java/ql/src/Security/CWE/CWE-113/ResponseSplitting.qhelp

```diff
@@ -5,16 +5,24 @@
 
 <overview>
 <p>Directly writing user input (for example, an HTTP request parameter) to an HTTP header
-can lead to an HTTP response-splitting vulnerability.
-If the user input includes blank lines in it, and if the servlet container does not itself
-escape the blank lines, then a remote user can cause the response to turn into two separate
-responses, one of which is controlled by the remote user.</p>
+can lead to an HTTP request-splitting or response-splitting vulnerability.</p>
+
+<p>HTTP Response-splitting can lead to vulnerabilities such as XSS, and cache poisoning.</p>
+<p>HTTP Request-splitting can allow an attacker to inject/smuggle an additional HTTP request into a client's outgoing socket connection.
```

Comment:
If "smuggle" in this context is a term of art that differs substantially from "inject", I'd go with "inject or smuggle". Otherwise, I think just "inject" is clear enough.

```suggestion
<p>HTTP request splitting can allow an attacker to inject an additional HTTP request into a client's outgoing socket connection.
```